grafana-plugin/src/containers/UserSettings/parts/tabs/SlackTab/SlackTab.tsx

```diff
@@ -32,7 +32,11 @@ export const SlackTab = () => {
 
           <Text type="secondary">
             More details in{' '}
-            <a href="https://grafana.com/docs/grafana-cloud/oncall/open-source/#slack-setup">
+            <a
+              href="https://grafana.com/docs/grafana-cloud/oncall/open-source/#slack-setup"
```

Comment:
same here, maybe we can extract this URL into a constant somewhere higher up and replace references to it?